cluster-autoscaler/hack/update-mods.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -o pipefail
+set -o nounset
+shopt -s lastpipe
+
+err_report() {
+    echo "Exited with error on line $1"
+}
+trap 'err_report $LINENO' ERR
+
+SCRIPT_NAME=$(basename "$0")
+K8S_FORK="kubernetes/kubernetes"
+# Update here when we need a new version of Kubernetes
+K8S_REV="ef7808fec57284582c7bf3fc834570e5769df83e"
+TARGET_MODULE=${TARGET_MODULE:-k8s.io/autoscaler/cluster-autoscaler}
+PRINT_HELP="false"
+UPDATE_GO_MOD="true"
+
+# Packages that are allowed to drift from k8s/k8s
+PERMITTED_PACKAGE_DIFFS=(
+    github.com/aws/aws-sdk-go
+    github.com/stretchr/testify
+)
+
+# Kubernetes staging repos: will need to be periodically updated
+# or update this script to check what is in k8s.io/kubernetes/stagin/src each time
+K8S_STAGING_MODULES=(
```

Comment:
You can parse those out of k8s `go.mod` file.

Reply:
Done